cmd/preview/deploy.go

```diff
@@ -263,3 +263,13 @@ func waitForResourcesToBeRunning(ctx context.Context, name, namespace string, t
 	}
 	return nil
 }
+
+func getExpandedName(name string) string {
+	username := okteto.GetUsername()
+	os.Setenv("OKTETO_USERNAME", username)
```

Comment:
this will set OKTETO_USERNAME in the env for the entire execution. Is that what we want? (if so, then I'd suggest we do that earlier, on a separate function, so we can use it in other places as well). 

IF not,  maybe we can use a library that only does the expansion for a string, without requiring it in the global env? (I think we use envsubst somewhere else?)

Reply:
I would go with option a)